[2023-07-22T05:54:11.876+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.create_table scheduled__2023-07-14T00:00:00+00:00 [queued]>
[2023-07-22T05:54:11.937+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.create_table scheduled__2023-07-14T00:00:00+00:00 [queued]>
[2023-07-22T05:54:11.937+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2023-07-22T05:54:11.937+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2023-07-22T05:54:11.938+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2023-07-22T05:54:12.017+0000] {taskinstance.py:1383} INFO - Executing <Task(PostgresOperator): create_table> on 2023-07-14 00:00:00+00:00
[2023-07-22T05:54:12.031+0000] {standard_task_runner.py:55} INFO - Started process 5154 to run task
[2023-07-22T05:54:12.060+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'user_processing', 'create_table', 'scheduled__2023-07-14T00:00:00+00:00', '--job-id', '38', '--raw', '--subdir', 'DAGS_FOLDER/user_processing.py', '--cfg-path', '/tmp/tmp4v42jtfu']
[2023-07-22T05:54:12.069+0000] {standard_task_runner.py:83} INFO - Job 38: Subtask create_table
[2023-07-22T05:54:12.359+0000] {task_command.py:376} INFO - Running <TaskInstance: user_processing.create_table scheduled__2023-07-14T00:00:00+00:00 [running]> on host ef71d3f52879
[2023-07-22T05:54:12.665+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=user_processing
AIRFLOW_CTX_TASK_ID=create_table
AIRFLOW_CTX_EXECUTION_DATE=2023-07-14T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-07-14T00:00:00+00:00
[2023-07-22T05:54:12.690+0000] {base.py:71} INFO - Using connection ID 'postgres' for task execution.
[2023-07-22T05:54:12.705+0000] {sql.py:315} INFO - Running statement: 
            CREATE TABLE IF NOT EXISTS users (
                firstname TEXT NOT NULL,
                lastname TEXT NOT NULL,
                country TEXT NOT NULL,
                username TEXT NOT NULL,
                password TEXT NOT NULL,
                email TEXT NOT NULL
            );
        , parameters: None
[2023-07-22T05:54:12.711+0000] {postgres.py:96} INFO - NOTICE:  relation "users" already exists, skipping

[2023-07-22T05:54:12.729+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=user_processing, task_id=create_table, execution_date=20230714T000000, start_date=20230722T055411, end_date=20230722T055412
[2023-07-22T05:54:12.894+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2023-07-22T05:54:13.092+0000] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-07-22T05:56:53.216+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.create_table scheduled__2023-07-14T00:00:00+00:00 [queued]>
[2023-07-22T05:56:53.239+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.create_table scheduled__2023-07-14T00:00:00+00:00 [queued]>
[2023-07-22T05:56:53.255+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2023-07-22T05:56:53.259+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2023-07-22T05:56:53.262+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2023-07-22T05:56:53.446+0000] {taskinstance.py:1383} INFO - Executing <Task(PostgresOperator): create_table> on 2023-07-14 00:00:00+00:00
[2023-07-22T05:56:53.488+0000] {standard_task_runner.py:55} INFO - Started process 5443 to run task
[2023-07-22T05:56:53.550+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'user_processing', 'create_table', 'scheduled__2023-07-14T00:00:00+00:00', '--job-id', '97', '--raw', '--subdir', 'DAGS_FOLDER/user_processing.py', '--cfg-path', '/tmp/tmppvuoftqk']
[2023-07-22T05:56:53.608+0000] {standard_task_runner.py:83} INFO - Job 97: Subtask create_table
[2023-07-22T05:56:54.006+0000] {task_command.py:376} INFO - Running <TaskInstance: user_processing.create_table scheduled__2023-07-14T00:00:00+00:00 [running]> on host ef71d3f52879
[2023-07-22T05:56:54.445+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=user_processing
AIRFLOW_CTX_TASK_ID=create_table
AIRFLOW_CTX_EXECUTION_DATE=2023-07-14T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-07-14T00:00:00+00:00
[2023-07-22T05:56:54.494+0000] {base.py:71} INFO - Using connection ID 'postgres' for task execution.
[2023-07-22T05:56:54.505+0000] {sql.py:315} INFO - Running statement: 
            CREATE TABLE IF NOT EXISTS users (
                firstname TEXT NOT NULL,
                lastname TEXT NOT NULL,
                country TEXT NOT NULL,
                username TEXT NOT NULL,
                password TEXT NOT NULL,
                email TEXT NOT NULL
            );
        , parameters: None
[2023-07-22T05:56:54.513+0000] {postgres.py:96} INFO - NOTICE:  relation "users" already exists, skipping

[2023-07-22T05:56:54.532+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=user_processing, task_id=create_table, execution_date=20230714T000000, start_date=20230722T055653, end_date=20230722T055654
[2023-07-22T05:56:54.652+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2023-07-22T05:56:54.748+0000] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
